packages/server/src/api/access-request/access-request.model.ts

```diff
@@ -53,6 +54,7 @@ export class AccessRequest extends BaseEntity {
   })
   whatYouDo!: string[];
 
+  @Index('access-request-sponsorName')
```

Comment:
I'm no so familiar with typeORM indices but @larskendall can you confirm if this annotation is even necessary if we are defining in the migration already (see below)?

Reply:
The migration was computed based on the definition of the models.  Removing the index ornamentation would break ORM/Schema consistency, but would not affect functionality of the code.  In performing some local testing, all tests continue to pass (with ornamentation removed, and migration .ts in place), and there didn't seem to be any significant performance changes.  Plus, removal of the indices post migration generation might result in removal of the indices if future ORM/schema changes are implemented.

Reply:
The decorator is necessary for TypeORM to generate schema changes. If we don't have them I think TypeORM will constantly try to remove those indexes any time you generate a new migration.